git_latexdiff_branch.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+repo_path=.
+old_main_file=main.tex
+new_main_file=main.tex
+temporary_dir=$repo_path/tmp
+mkdir $temporary_dir
```

Comment:
Consider adding this before mkdir to help new users and make the need for an argument easier to spot:
```
if [ -z "$1" ]; then
  echo "Usage: $0 git_branch_or_commit" >&2
  exit 1
fi
```